### ISSUE TITLE
RPPD not recognized on n-well (alpha LVS)

### ISSUE THREAD

**Issue by bmurmann** [IHP-GmbH/IHP-Open-PDK#90]
Title should be self-explanatory. The derivation file for RPPD excludes n-well as a valid region for RPPD (which I don't think is correct).

**Comment by FaragElsayed2** [comment]
Hi Dr. @bmurmann 

Is this applicable only to `rppd`, or does it apply to all poly resistors `(rhigh, rppd, rsil)`?

**Comment by bmurmann** [comment]
Yes, I believe all poly resistors should be allowed on n-well, but you may want to double check with @KrzysztofHerman.

**Comment by KrzysztofHerman** [comment]
@FaragElsayed2 @bmurmann I confirm, you are free to use all resistor in both regions: `p-sub` and `n-well`.

**Comment by sergeiandreyev** [comment]
Hi all, as I understand, this should be fixed in the latest update of LVS deck..
@bmurmann, could you please confirm?

**Comment by bmurmann** [comment]
@sergeiandreyev Yes, confirmed.
